### ISSUE TITLE
Exception in Rollbar when using file config

### ISSUE THREAD

**Issue by willmcgugan** [rollbar/pyrollbar#277]
Rollbar raises the following exception when configured from logging.fileConfig:

```
  File "rollbarbug.py", line 7, in <module>
    log.error('test')
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/logging/__init__.py", line 1361, in error
    self._log(ERROR, msg, args, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/logging/__init__.py", line 1468, in _log
    self.handle(record)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/logging/__init__.py", line 1478, in handle
    self.callHandlers(record)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/logging/__init__.py", line 1540, in callHandlers
    hdlr.handle(record)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/logging/__init__.py", line 854, in handle
    self.emit(record)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/rollbar/logger.py", line 108, in emit
    if record.levelno < self.notify_level:
TypeError: '<' not supported between instances of 'int' and 'str'
```

The following code will demonstrate this:

```python
import logging
import logging.config

logging.config.fileConfig('./logging.conf')

log = logging.getLogger('web')
log.error('test')
```

**logging.conf** is as follows:

```
[loggers]
keys=root,web

[handlers]
keys=console,null,rollbar

[formatters]
keys=simple

[logger_root]
level=NOTSET
handlers=null

[logger_web]
qualname=web
level=DEBUG
handlers=console,rollbar

[handler_null]
class=NullHandler
level=NOTSET
formatter=simple
args=()

[handler_console]
class=StreamHandler
level=DEBUG
formatter=simple
args=(sys.stdout,)

[handler_rollbar]
level=ERROR
class=rollbar.logger.RollbarHandler
args=()

[formatter_simple]
format=%(name)s: %(message)s
datefmt=[%d/%b/%Y %H:%M:%S]
```

This is on Python3.6. It didn't occur on Python2.7 because it allows comparisons between different types, but it also wouldn't have been working as intended...

**Comment by jessewgibbs** [comment]
Hi Will,
Sorry for the slow response to your issue.  The primary maintainer of pyrollbar is out at the moment, and no one else on our team has had a chance to look at it.  We'll aim to review this and respond soon!

**Comment by brianr** [comment]
Thanks to @coryvirok for jumping on this. We've got the fix released in 0.14.2.
